scripts/Install-ExchPrerequisites.ps1

```diff
@@ -0,0 +1,59 @@
+﻿[CmdletBinding()]
+
+param(
+
+    [Parameter(Mandatory=$true)]
+    [string]
+    $ExchangeServerVersion
+
+)
+
+try {
+    Start-Transcript -Path C:\cfn\log\Install-ExchPrerequisites.ps1.txt -Append
+    $ErrorActionPreference = "Stop"
+    
+    $Retries = 0
+    $Installed = $false
+    
+    while (($Retries -lt 4) -and (!$Installed)) {
```

Comment:
Take a look at the updated version of the script this is based on from the SQL Quick Start. It's been enhanced a bit to make it more readable, etc.